run_dir/static/js/running_notes.js

```diff
@@ -1,12 +1,12 @@
 $(function(){
-  /*  Once projects are moved to new running notes db
+
   if(typeof project!=='undefined'){
       $.getJSON('/api/v1/latest_sticky_run_note/'+project, function (data) { 
         //latest_sticky_note
         let sticky_run_note = data 
         let date = Object.keys(sticky_run_note)[0]
```

Comment:
Is the output from `Object.keys` always sorted or is it just one key?

Reply:
Its just the one key.